packages/bigtable/src/mutation.js

```diff
@@ -80,6 +80,10 @@ Mutation.convertFromBytes = function(bytes) {
  * @return {buffer}
  */
 Mutation.convertToBytes = function(data) {
+  if (data instanceof Buffer) {
```

Comment:
I think I was mixing up some edge cases with comparing buffers without the help of modules. If `instanceof` is working, let's stick with that 👍